totalRP3/core/impl/slash.lua

```diff
@@ -34,6 +34,7 @@ TRP3_API.slash = {}
 --*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
 
 local COMMANDS = {};
+local CONDITIONALS = {};

```

Comment:
It has `CAPITAL_CASING` as if it's a constant, but with functions like `registerConditional` this isn't really a constant.

Reply:
This is consistent with other similar variables.